rmf_building_map_tools/building_map_server/building_map_server.py

```diff
@@ -278,7 +278,7 @@ def lift_msg(self, lift):
         # transformation is already done in Lift class
         msg.ref_x, msg.ref_y = lift.x, lift.y
         msg.name = str(lift.name)
-        msg.levels = lift.level_names
+        msg.levels = str(lift.level_names)
```

Comment:
This doesn't seem correct to me... I expect `lift.level_names` to be a list, so we would want to convert each element to a `str`, not the entire list into one `str`, e.g.:

```suggestion
        msg.levels = str(level_name) for level_name in lift.level_names
```

I'd appreciate if you can test this out on a multi-level building and let us know how it turns out.

Reply:
Noted. Thank you for the suggestion.

Will make the necessary changes and test. 

Will update after I have tested.

Reply:
Hi @mxgrey 

I have made the changes. Tested as well by duplicating the first map and naming it `2`. 

Able to view both maps `1` and `2` in RViz.